doc/classes/Engine.xml

```diff
@@ -67,12 +71,14 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether or not the editor is running.
 			</description>
 		</method>
-		<method name="is_in_fixed_frame" qualifiers="const">
+		<method name="is_in_physics_frame" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether or not the game is inside the fixed process and physics phase of the game loop.
```

Comment:
Per writing guidelines, boolean methods/members prefer "[code]true[/code]" over "whether or not".

Returns [code]true[/code] if the game is in the fixed process or physics phase of the game loop.